### PR TITLE
Updating Angela with SiQi + update intro

### PR DESCRIPTION
**Pull request by knmcguire**
* Updating angela speaker with SiQi
* Updating text of the introduction

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -208,7 +211,7 @@ <h2 class="title is-3">Schedule</h2>
         <li>15:30 - Coffee break </li>
         <li>16:00 - Crazyswarm2 Tutorial </li>
         <li>16:30 - Crazychoir Tutorial </li>
-        <li>17:00 - Angela Schoelig </li>
+        <li>17:00 - SiQi Zhou (on behalve of Angela Schoellig) </li>
```
Comment (whoenig) [inline index.html]: behalf

**Review by whoenig — CHANGES_REQUESTED** [review]
One spelling mistake, otherwise it looks good.

File: index.html (hunk 1 above)
Comment (knmcguire) [inline index.html]:
```suggestion
        <li>17:00 - SiQi Zhou (on behalf of Angela Schoellig) </li>
```

**Comment by knmcguire** [comment]
Thanks! haha I did it correct the other time around. I've fixed it so I'll merge

**Pull request merged**